h/static/scripts/controllers/input-autofocus-controller.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const Controller = require('../base/controller');
+
+function isWordChar(event) {
+  return event.key.match(/^\w$/) && !event.ctrlKey && !event.altKey;
+}
+
+/**
+ * Automatically focuses an input field when the user presses a letter, number
+ * or backspace if no other element on the page has keyboard focus. The field's
+ * focus can also be blurred by pressing Escape.
```

Comment:
I think what will happen is that whichever one was initialized first will have its event handler run when a key is typed, that will cause a change in focus and then the second event handler will do nothing because the body no longer has focus.